cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java

```diff
@@ -801,7 +803,14 @@ void scheduleFilteredBanners() {
         continue;
       }
 
-      banner.setScheduled();
+      boolean isEveryTrigger = banner.getFrequency() == BannerFrequency.Every_Trigger && banner.getTriggerType() == BannerTriggerType.Conditions;
+      if (!checkIsEveryTrigger(banner, isEveryTrigger)) {
+        continue;
+      }
+
+      if (!isEveryTrigger) {
```

Comment:
not sure if this makes sense, because if isEveryTrigger is true, then we will never schedule the banner?